build/zork.tcl

```diff
@@ -16,7 +16,6 @@ expect -timeout 6000 "Job XXFILE interrupted: .VALUE;"
 type "\033p"
 expect -timeout 500 ":KILL"
 
-respond "*"  ":xxfile lcf;TTY:_lcf;zork xxfile\r"
```

Comment:
This looks strange.  Did you remove the second :xxfile?

Reply:
oops. Not sure how that happened. I just wanted to change from writing to log to write to tty:. Added that line back.